apiserver/common/errors.go

```diff
@@ -91,6 +91,10 @@ func IsUpgradeInProgressError(err error) bool {
 // the user) has been migrated to a different controller.
 type RedirectError struct {
 	// Servers holds the sets of addresses of the redirected servers.
+	// TODO (manadart 2019-11-08): Change this to be either MachineHostPorts
```

Comment:
This seems OK to omit for now. These addresses are parsed rather than resolved by lookup.